README.md

```diff
@@ -28,10 +45,11 @@ jobs:
     runs-on: ubuntu-latest # only linux supported at present
     steps:
       - uses: actions/checkout@v1
-      - uses: w3c/respec-w3c-auto-publish@master # use the action
+      - uses: w3c/respec-w3c-auto-publish@v1 # use the action
         with:
           ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
-          ECHIDNA_MANIFEST_URL: "https://w3c.github.io/gamepad/W3CTRMANIFEST"
+          ECHIDNA_MANIFEST_URL: "https://w3c.github.io/your_spec_repo/ECHIDNA"
+          # Please use the URL that's appropriate for your working group!
```

Comment:
We should hunt these down... there is probably only like 5-10 URLs.

Reply:
Filed https://github.com/w3c/respec-w3c-auto-publish/issues/12